app/backend/approaches/MathTutor.py

```diff
@@ -40,7 +40,7 @@
 # openai.api_key = azure_openai_service_key
```

Comment:
change the name and it's refernces to Mathassistant. It will be hard to change later on.